### PR TITLE
Set base title from default config.

### PR DESCRIPTION
**Pull request by lukekhamilton**
Minor change to set base title from default config.

### CODE REVIEW

**Comment by lirantal** [comment]
LGTM

**Comment by codydaig** [comment]
+1

File: config/env/development.js [hunk 1]
```diff
@@ -18,7 +20,7 @@ module.exports = {
 		}
 	},
 	app: {
-		title: 'MEAN.JS - Development Environment'
+        title: defaultEnvConfig.app.title + ' - Development Environment'
```
Comment (rhutchison) [inline config/env/development.js]: Fix indent

Comment (ilanbiala) [inline config/env/development.js]: @lukekhamilton can you fix this and we'll merge.

**Comment by ilanbiala** [comment]
@lukekhamilton will you be able to fix the indentation in the next few hours or so?

**Comment by codydaig** [comment]
@ilanbiala Just in case @lukekhamilton doesn't get around too it, I am going to submit a new one as soon as I'm off the train.

**Comment by ilanbiala** [comment]
Great, we'll merge whichever happens first.

**Pull request closed without merging**